lib/rubocop/cop/packs/root_namespace_is_pack_name.rb

```diff
@@ -34,8 +34,10 @@ def on_new_investigation
           relative_filepath = absolute_filepath.relative_path_from(Pathname.pwd)
           relative_filename = relative_filepath.to_s
 
-          # This cop only works for files ruby files in `app`
-          return if !relative_filename.include?('app/') || relative_filepath.extname != '.rb'
+          # Only enforce this cop for ruby files
+          return if relative_filepath.extname != '.rb'
+          # Only enforce this cop for non-spec files in the app/ directory
+          return if !relative_filename.include?('app/') || relative_filename.include?('spec/')
```

Comment:
Do you think three guard statements would be easier to read?
```
return if relative_filepath.extname != '.rb'
return if relative_filename.include?('spec/')
return if !relative_filename.include?('app/')
```

Reply:
Hah! I actually had those same lines in a previous version. I'll add them back.